web/src/compose_setup.js

```diff
@@ -88,6 +94,10 @@ export function initialize() {
         } else {
             $(".add-poll").parent().removeClass("disabled-on-hover");
         }
+
+        // show banner if the option to split is enabled
+        // and the message contains the split delimiter
+        debounced_update_split_message_banner();
```

Comment:
Might be a better idea to call this only if splitting messages is enabled; less pointless debouncing on every keypress

Reply:
The function actually checks whether the message will split before doing anything with the banner.

Reply:
Yeah that's true, but debouncing even when it leads to nothing on finally executing the function is still unnecessary overhead that should be eliminated

Reply:
Sure, I'll remove it